src/os/bluestore/BlueStore.cc

```diff
@@ -810,10 +812,11 @@ void BlueStore::handle_conf_change(const struct md_config_t *conf,
 	     << bluestore_blob_t::get_csum_type_string(csum_type)
 	     << dendl;
   }
-  if (changed.count("bluestore_min_alloc_size")) {
-    min_alloc_size = g_conf->bluestore_min_alloc_size;
-    dout(10) << __func__ << " min_alloc_size 0x" << std::hex << min_alloc_size
-	     << std::dec << dendl;
+
+  if (changed.count("bluestore_min_alloc_size") ||
+      changed.count("bluestore_min_alloc_size_hdd") ||
+      changed.count("bluestore_min_alloc_size_ssd")) {
+    _set_min_alloc();
   }
```

Comment:
Let's drop this section, actually, since it is probably unsafe to change this setting on a running system.